dotnet/Templates/Microsoft.tvOS.Templates/tvos/.template.config/template.json

```diff
@@ -31,10 +31,10 @@
     },
     "minOSVersion": {
       "type": "parameter",
-      "description": "Overrides MinimumOSVersion in the Info.plist",
+      "description": "Overrides SupportedOSPlatformVersion in the project file",
       "replaces": "minOSVersion",
       "datatype": "string",
-      "defaultValue": "11.2"
+      "defaultValue": "14.0"
```

Comment:
why ? it's still `11.2` for iOS
which I assume is to use the new `UIScene` API as they are newer than the `10.0` minimum ?

Reply:
I'm not sure what happened here 🤔